docs: draft RFC-0360: NFT sale via Mimblewimble invoice

philipr-za opened the pull request:
Description
---
This PR provides a draft RFC for the process to negotiate an NFT sale as an Invoice. This is where the initiator of the transaction requests the payment amount from the counterparty.

Motivation and Context
---

How Has This Been Tested?
---

delta1 commented on RFC/src/RFC-0360_NFTInvoices.md:
```diff
@@ -0,0 +1,210 @@
+
+## Description
+The standard interactive [Mimblewimble] transaction is usually initiated by a party, the Sender, that is going to send 
+funds to a Receiver. The Sender selects a number of input UTXO's from which the sent funds and fees will come from, they
```
```suggestion
funds to a Receiver. The Sender selects a number of input UTXOs from which the sent funds and fees will come from, they
```

delta1 commented on RFC/src/RFC-0360_NFTInvoices.md:
```diff
@@ -0,0 +1,210 @@
+## Description
+The standard interactive [Mimblewimble] transaction is usually initiated by a party, the Sender, that is going to send 
+funds to a Receiver. The Sender selects a number of input UTXO's from which the sent funds and fees will come from, they
+build a change UTXO to receive any remaining funds and caluculate the excess of these inputs and change output. The 
```
```suggestion
build a change UTXO to receive any remaining funds and calculate the excess of these inputs and change output. The 
```

delta1 commented on RFC/src/RFC-0360_NFTInvoices.md:
```diff
@@ -0,0 +1,210 @@
+build a change UTXO to receive any remaining funds and caluculate the excess of these inputs and change output. The 
+Receiver is then told how many Tari they will receive and provides the aggregated public excess and public nonce of the 
+Sender's portion of the transaction signature. The Receiver builds a commitment with the specified value and their 
+private spending key and then provide their partial signature for their UTXO and the excess. This gets sent back to the 
```
```suggestion
private spending key and then provides their partial signature for their UTXO and the excess. This gets sent back to the 
```

delta1 commented on RFC/src/RFC-0360_NFTInvoices.md:
```diff
@@ -0,0 +1,210 @@
+|--------------------------------------         |-----------------------------------------------------------------------------------------------------------|
+| \\( C_{St} = 0 \cdot H  + k_{St} \cdot G \\)  | Select the Seller's token UTXO to be offered                                                              |
+| \\( C_{Sp} = v_p \cdot H + k_{Sp} \cdot G \\) | Choose a value to be requested from Buyer and select a spending key for the payment UTXO to be received   |
+| \\( R_S = r_S \cdot G) \\)                    | Choose a nonce                                                                                            |
```
I think there's a stray bracket here 

```suggestion
| \\( R_S = r_S \cdot G \\)                    | Choose a nonce                                                                                            |
```

delta1 reviewed (commented):
Looks good so far, just a few minor typos

philipr-za committed:
docs: draft RFC-0360: NFT sale via Mimblewimble invoice

This PR provides a draft RFC for the process to negotiate an NFT sale as an Invoice. This is where the initiator of the transaction requests the payment amount from the counterparty.

stringhandler reviewed (approved):
Looks good. There may be some details that crop up during implementation, but this seems solid to me

aviator-app[bot] committed:
Merge branch 'development' into philip-invoices-rfc
